src/NHibernate.Test/ReadOnly/TextHolder.cs

```diff
@@ -10,7 +10,7 @@ public class TextHolder
 		/// </summary>
 		public static bool SupportedForDialect(Dialect.Dialect dialect)
 		{
-			return !(dialect is FirebirdDialect || dialect is Oracle8iDialect);
+			return !(dialect is FirebirdDialect || dialect is Oracle8iDialect || dialect is MsSqlCeDialect);
```

Comment:
Here too, I would rather have a `SupportsTextBlobType` feature. This blacklisting may miss a bunch of supported db we are currently not testing. And when those `text`/ `ntext` types will start being dropped from newer version of SQL Server, the blacklisting will start to be cumbersome. (Granted, this looks like it may never happen, `ntext` and `text` are in the ["will be removed in an undetermined future version"](https://docs.microsoft.com/en-us/sql/database-engine/deprecated-database-engine-features-in-sql-server-2016#features-not-supported-in-a-future-version-of-sql-server) list since SQL 2005 till SQL 2016. Still waiting to see them in the "will be removed in next version list".)

By the way, character types looks like being another muddy place a bit alike dates types^^. (Even worse indeed.)